src/libraries.sh

```diff
@@ -133,35 +135,10 @@ __INTERNAL_rlLibrarySearchInRoot(){
 
   rlLogDebug "rlImport: Trying root: [$BEAKERLIB_LIBRARY_PATH]"
 
-  local CANDIDATE="$BEAKERLIB_LIBRARY_PATH/$COMPONENT/Library/$LIBRARY/lib.sh"
-  if [ -f "$CANDIDATE" ]
-  then
-    LIBFILE="$CANDIDATE"
-    return
-  fi
-
-  local CANDIDATE="$( echo $BEAKERLIB_LIBRARY_PATH/*/$COMPONENT/Library/$LIBRARY/lib.sh )"
-  if [ -f "$CANDIDATE" ]
-  then
-    LIBFILE="$CANDIDATE"
-    return
-  fi
-
-  local CANDIDATE="$BEAKERLIB_LIBRARY_PATH/$COMPONENT/$LIBRARY/lib.sh"
-  if [ -f "$CANDIDATE" ]
-  then
-    LIBFILE="$CANDIDATE"
-    return
-  fi
-
-  local CANDIDATE="$BEAKERLIB_LIBRARY_PATH/libs/$COMPONENT/$LIBRARY/lib.sh"
-  if [ -f "$CANDIDATE" ]
-  then
-    LIBFILE="$CANDIDATE"
-    return
-  fi
+  LIBFILE="$(__INTERNAL_rlLibrarySearchInDir "$BEAKERLIB_LIBRARY_PATH" "$COMPONENT" "$LIBRARY")" && return
 
   rlLogDebug "rlImport: Library not found in $BEAKERLIB_LIBRARY_PATH"
```

Comment:
Doesn't this duplicate the same warning as in function `__INTERNAL_rlLibrarySearchInDir`?

Reply:
your right